awe_components/setup/data.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python3
 # Copyright 2022, Educational Testing Service
 
+# TODO: using numpy>1.21.4 seems to cause errors when installing from awe_workbench.
```

Comment:
The version lock is `1.26.4`. Is this just a typo?

Reply:
Is this the correct spot for this comment? I thought it had to do with the model we are using. So this comment ought to be placed where the model is loaded in. A quick search in the codebase makes me think it belongs in `awe_components/wordprobs/wordProbabilityInContext.py`; however, this may not be the correct spot for it.

Reply:
> The version lock is `1.26.4`. Is this just a typo?

Yup! I'll fix that.

> Is this the correct spot for this comment? I thought it had to do with the model we are using. So this comment ought to be placed where the model is loaded in. A quick search in the codebase makes me think it belongs in `awe_components/wordprobs/wordProbabilityInContext.py`; however, this may not be the correct spot for it.

From what I remember about the errors I was getting, it was referencing the spacy.cli.download import in `data.py`. I'll look at `awe_components/wordprobs/wordProbabilityInContext.py` nonetheless, but I think my comment is in the right place.